Probesys/bileto
# new: Allow to transfer tickets

## Related issue(s)



https://github.com/Probesys/bileto/issues/170

## How to test manually



1. Create a ticket in organization A
2. Assign an agent, observers and a team which have access only to this organization
3. Assign a contract to this ticket
4. Spent time on this ticket and make sure to account it for the contract
5. Create an ongoing contract in organization B
6. Transfer the ticket to organization B
7. Verify that the actors and the contract have been removed + that spent time is now unaccounted

## Checklist
